docs/optimizing/advanced_retrieval/How to get +15% hit_rate improvement using DeepMemory.ipynb

```diff
@@ -0,0 +1,749 @@
+{
```

Comment:
<p>some nitpicks and mispelling</p><p>like <code>relevence</code> to relevance</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/8625/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>